### ISSUE TITLE
phone number verification process

### ISSUE THREAD

**Issue by mickychen0524** [firebase/FirebaseUI-iOS#338]
google verify window never dismiss after choose squares

do I have to use any function for AuthUIDelegate to dismiss?

**Comment by mickychen0524** [comment]
btw, it seems to be showing on simulator only, I upgraded my device to iOS 11 and can't see this process

**Comment by yramanchuk** [comment]
@mickychen0524 Can you please provide required information? Just read information when you hit "new issue").

> google verify window never dismiss after choose squares

Do you mean that `SFSafariViewController`(SFSVC) with reCAPTCHA is not closed after you select vehicles or just another SFSVC is shown? When reCAPTCHA was not solved properly you will be asked to solve another one and it will happen until you solve it properly. Only after that SFSVC will be closed.

> btw, it seems to be showing on simulator only, I upgraded my device to iOS 11 and can't see this process

Please notice that reCAPTCHA is shown only in simulator and if you properly configured Push Notifications it should not appear on device if device verification flow started properly (read as: reCAPTCHA should not be shown on device).

> do I have to use any function for AuthUIDelegate to dismiss?

No, reCAPTCHA will be hidden automatically after user solved it.

@mickychen0524 Please let me know if that information helped you.

**Comment by mickychen0524** [comment]
@yramanchuk thanks for your explanation, it doesn't show on device
